pkg/occlient/occlient.go

```diff
@@ -1096,12 +1096,23 @@ func (c *Client) Delete(labels map[string]string) error {
 		}
 	}
 	// PersistentVolumeClaim
-	log.Debugf("Deleting PersistentVolumeClaims")
+	log.Debug("Deleting PersistentVolumeClaims")
 	err = c.kubeClient.CoreV1().PersistentVolumeClaims(c.namespace).DeleteCollection(&metav1.DeleteOptions{}, metav1.ListOptions{LabelSelector: selector})
 	if err != nil {
 		errorList = append(errorList, "unable to delete volume")
 	}
 
+	// Service Instance
+	log.Debug("Deleting Service Instance")
```

Comment:
Add more comments here of what's happening... It's a lot of long lines / code, we should add commenting around htere of what's happening.

Reply:
Added :)